esm_runscripts/compute.py

```diff
@@ -244,46 +244,73 @@ def _write_finalized_config(config):
 
     @staticmethod
     def copy_tools_to_thisrun(config):
```

Comment:
One thing I never understood about this method: We copy everything to the experiment. That is nice. However, I think the actual yaml files and namelists are still read from wherever the `.esmtoolsrc` file points to?

Reply:
Yes, this thing was doing virtually nothing, because it would only update the `namelists` and `esm_tools` when using  `-U` but not the runscript, and it is only the runscript that is actually used. That is why I include the runscript. I left the rest as it was cause I thought there should be a reason for that, but if it is a deprecated feature maybe we can think of getting rid of the `namelist` and `esm_tools` copying.